java-sdk-example/build.gradle

```diff
@@ -13,10 +13,30 @@ repositories {
 }
 
 dependencies {
+    testImplementation 'io.github.openfeign:feign-core:10.11'
+    testImplementation 'io.github.openfeign:feign-jackson:10.11'
+    testImplementation 'io.github.openfeign:feign-slf4j:10.11'
+    testImplementation 'io.github.openfeign.form:feign-form:3.8.0'
+    testImplementation 'io.github.openfeign:feign-okhttp:10.11'
+    testImplementation 'com.fasterxml.jackson.core:jackson-core:2.10.3'
+    testImplementation 'com.fasterxml.jackson.core:jackson-annotations:2.10.3'
+    testImplementation 'com.fasterxml.jackson.core:jackson-databind:2.10.3'
+    testImplementation 'com.fasterxml.jackson.datatype:jackson-datatype-jsr310:2.10.3'
+
+    testImplementation 'org.openapitools:jackson-databind-nullable:0.2.2'
+
+    testImplementation 'com.github.scribejava:scribejava-core:8.0.0'
+    testImplementation 'jakarta.annotation:jakarta.annotation-api:1.3.5'
+
+    testImplementation 'ch.qos.logback:logback-classic:1.2.10'
+    // TODO: Remove all dependencies above this line, only added when testing locally.
```

Comment:
@Freezerburn : I must have missed the TODO earlier. If we're sure that we won't be required to include these when using the sdk, the rest of the code change needed as a part of the switch to feign looks good to me. However, why were these needed for testing locally in the first place? Shouldn't they have been automatically available as they're dependency of the SDK?

To test it one step further, let's publish the SDK snapshot and use that instead of the local copy. That way, it's close going to be closer to the actual implementation, and also, we can be confident that the snapshot publication is working fine.